ai-ml/nvidia-triton-server/addons.tf

```diff
@@ -140,6 +146,20 @@ module "eks_blueprints_addons" {
     ],
   }
 
+  helm_releases = {
+    "prometheus-adapter" = {
+      repository = "https://prometheus-community.github.io/helm-charts"
+      chart      = "prometheus-adapter"
+      namespace  = module.eks_blueprints_addons.kube_prometheus_stack.namespace
+      version    = "4.10.0"
+      values = [
+        templatefile(
+          "${path.module}/helm-values/prometheus-adapter.yaml", {}
```

Comment:
Considering that the custom metric for triton-vLLM will be different with nim disabled, we should rename this adapter to something like `prometheus-adpater-nim.yaml`. Idea is only if nim is enabled this particular config will be used.

Reply:
got your idea. but honestly i don't think it deserve to make it too complex. 
considering there will be cases including 1/enable_triton only; 2. enable_nim only; 3. enable both; then it falls into some difficult cases you have to handle.

current settings/configs dosen't introduce any side effect even if only triton-server enabled; if you have the customized metrics for tritton-vllm later, you can just add those metrics on top of existing one. 
@ratnopamc Does it make sense to you?

Reply:
as aligned offline, will keep the prometheus-adapter addon as it's common for both triton and nim pattern. inject the prometheus namespace into the values with a variable